src/firefly/js/ui/NaifidPanelWorker.js

```diff
@@ -93,7 +93,7 @@ function resolveObject(objName) {
     }
 
     let {p}= makeResolverPromise(objName);
-    p= p.then( (results) =>
+    p.then( (results) =>
```

Comment:
My mistake.  Trey corrected me.  You had it right before.
It was just awkward to see it written that way.  Please revert this change.

Reply:
Completed, Thanks!!